kong/plugins/jwt/schema.lua

```diff
@@ -28,6 +28,7 @@ return {
     anonymous = {type = "string", default = "", func = check_user},
     run_on_preflight = {type = "boolean", default = true},
     maximum_expiration = {type = "number", default = 0, func = check_positive},
+	inject_claims = {type = "array", default = {}},
```

Comment:
style: please watch the indentation here :)

Reply:
Will fix in the next commit